modules/custom/az_finder/src/Plugin/views/exposed_form/QuickstartExposedFilters.php

```diff
@@ -0,0 +1,226 @@
+      unset($form['bef']['general']['reset_button_always_show']);
+    }
+    if (isset($form['bef']['general']['reset_button_label'])) {
+      $form['bef']['general']['reset_button_settings']['reset_button_label'] = $form['bef']['general']['reset_button_label'];
+      unset($form['bef']['general']['reset_button_label']);
+    }
+    $form['bef']['general']['reset_button_settings']['reset_button_position'] = [
+      '#type' => 'select',
+      '#title' => $this->t('Reset Button Position'),
+      '#options' => [
+        'top' => $this->t('Top'),
+        'bottom' => $this->t('Bottom'),
+      ],
+      '#default_value' => $this->options['reset_button_position'] ?? 'bottom',
+      '#description' => $this->t('Select where to place the reset button in the form.'),
+    ];
+    $form['bef']['general']['reset_button_settings']['reset_button_counter'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Show active filter counter'),
+      '#description' => $this->t('Show a counter of active filters next to the reset button.'),
+      '#default_value' => $this->options['reset_button_counter'] ?? FALSE,
+    ];
+    $form['bef']['general']['skip_link'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Add skip link to the top of the view'),
+      '#description' => $this->t('Add a skip link to the top of the view to allow keyboard users to skip to the search and filter form.'),
+      '#default_value' => $this->options['skip_link'] ?? FALSE,
+    ];
+
+    $form['bef']['general']['skip_link_settings'] = [
+      '#type' => 'details',
+      '#title' => $this->t('Skip Link Settings'),
+      '#open' => TRUE,
+      '#states' => [
+        'visible' => [
+          ':input[name="exposed_form_options[bef][general][skip_link]"]' => ['checked' => TRUE],
+        ],
+      ],
+    ];
+    $form['bef']['general']['skip_link_settings']['skip_link_text'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Text'),
+      '#description' => $this->t('The text to display for the skip link.'),
+      '#default_value' => $this->options['skip_link_text'] ?? $this->t('Skip to search and filter'),
+    ];
+    $form['bef']['general']['skip_link_settings']['skip_link_id'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('ID/Anchor'),
+      '#default_value' => $this->options['skip_link_id'] ?? 'search-filter',
+      '#description' => $this->t('The ID or anchor to link to within the view.'),
+    ];
+    $form['bef']['general']['orientation'] = [
+      '#type' => 'radios',
+      '#title' => $this->t('Orientation'),
+      '#description' => $this->t('The orientation of the filters within the exposed form.'),
+      '#options' => [
+        'horizontal' => $this->t('Horizontal'),
+        'vertical' => $this->t('Vertical'),
+      ],
+      '#default_value' => $this->options['orientation'] ?? FALSE,
```

Comment:
Looking to have the default orientation be vertical.

```suggestion
      '#default_value' => $this->options['orientation'] ?? TRUE,
```